tools/downloader/downloader.py

```diff
@@ -225,6 +235,11 @@ def main():
             output.mkdir(parents=True, exist_ok=True)
 
             for model_file in model.files:
+                # Download only files with specific precision.
+                prefix = str(model_file.name)[:str(model_file.name).find('/')]
```

Comment:
The logic that decides which file belongs to which precision should be located in `common`, and in fact there essentially already _is_ such logic there. You just need to change the data that `common.load_topologies` returns to include this information.

On the downloader level, the logic should be more high level: "if this file belongs to a precision, and the user requested this precision, then download it".

Reply:
Opened a separate PR https://github.com/opencv/open_model_zoo/pull/384